nifi-mock/pom.xml

```
@@ -66,8 +66,8 @@
         <dependency>
             <!-- Dependency marked as compile, not test, because we have assertion
             methods in our MockSession & MockFlowFile -->
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
```

To be more precise in dependency declarations, this should be `junit-jupiter-api` instead of `junit-jupiter-engine`.
```suggestion
            <artifactId>junit-jupiter-api</artifactId>
```